apps/telegraf/src/app/telegraf-main/components/main-tabs/main-tabs.component.html

```diff
@@ -0,0 +1,40 @@
+<!--<ul class="nav nav-tabs">
+  <li class="nav-item" [ngClass] = "{active: chosenList === 'najcitanije'}">
+    <a class="nav-link" (click) = "onClick('najcitanije')" >Najčitanije</a>
+  </li>
+  <li class="nav-item" [ngClass] = "{active: chosenList === 'komentari'}">
+    <a class="nav-link" (click) = "onClick('komentari')">Komentari</a>
```

Comment:
```suggestion
 <a class="nav-link" (click) = "onClick(yourConstants.komentari)">Komentari</a>
```


import { MY_CONSTANT} from '@path/.../constants';

@Component({
  // ...
})
export class MyTestComponent implements OnInit {

  readonly yourConstants = MY_CONSTANT;

  // ...
}